recipes/libcurl/all/conanfile.py

```diff
@@ -171,9 +171,6 @@ def build_requirements(self):
             self.build_requires("msys2/20200517")
         elif self._is_win_x_android:
             self.build_requires("ninja/1.10.2")
-        elif not tools.os_info.is_windows:
-            self.build_requires("libtool/2.4.6")
-            self.build_requires("pkgconf/1.7.3")
```

Comment:
it breaks native builds !

Reply:
In the beginning i kept the build requirements with a test for cross building
```
 elif not tools.os_info.is_windows and not tools.cross_building(self.settings):
```

The suggested recipes for autotools (mpc, libatomic_ops, libev) in https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md#how-to-provide-a-good-recipe do not add build_requirements for libtool or pkgconf. This was the reason I completely removed the build dependencies. I guess it works on conan because the conanio/gcc* packages all bundle the programs themselves.

Reply:
>In the beginning i kept the build requirements with a test for cross building
>
> ```
> elif not tools.os_info.is_windows and not tools.cross_building(self.settings):
> ```

Where? I don't see this line if your modifications

> The suggested recipes for autotools (mpc, libatomic_ops, libev) in https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md#how-to-provide-a-good-recipe do not add build_requirements for libtool or pkgconf. This was the reason I completely removed the build dependencies. I guess it works on conan because the conanio/gcc* packages all bundle the programs themselves.

What dou you mean? mpc doesn't need autotools, because it directly calls configure script. It doesn't need pkgconf because I guess that its dependencies are discovered in another way.

Reply:
I have added them again and prevent the build_requirements when cross_build is done

Reply:
Are you using two profiles?

Reply:
We tried building libcurl using the container https://hub.docker.com/r/conanio/gcc7-armv7hf and could not build it in this container.
We tried using profiles for the host and the target (see issue https://github.com/conan-io/conan-center-index/issues/4897). We found that removing the build dependencies and using the docker container we can simply build the library and compile our project.

Reply:
So it might be an issue in pkconf recipe, eventuall coming from meson, not in libcurl.

Reply:
We have tried again with building with profiles in #4897 and now it builds using (https://docs.conan.io/en/latest/systems_cross_building/cross_building.html#cross-building). Thanks for looking into the PR. I'm closing the PR because its not needed.